ADVANCED-USAGE-README.md

```diff
@@ -0,0 +1,60 @@
+# PatternFly React
+This README covers advanced usage topics for PatternFly React users.  
+
+### Table of contents
+1. [Applying Overpass Font](#Applying-Overpass-font)
+2. [Tree shaking](#Tree-shaking)
+
+## Applying Overpass font
+If you would like to use Overpass instead of the Red Hat font, simply add the class `.pf-m-overpass-font` to an element that wraps your application (ideally `<html>` or `<body>`) to adopt the CSS changes that would allow opting in to the Overpass font.
+
+
+## Tree shaking
+
+Ensure optimization.sideEffects is set to true within your Webpack config:
+```JS
+optimization: {
+  sideEffects: true
+}
+```
+
+Use ESM module imports to enable tree shaking with no additional setup required.
+```JS
+import { TimesIcon } from '@patternfly/react-icons';
```

Comment:
is this ok, i thought we were directly importing from the file not the barrel

Reply:
Consumers are free to use this syntax. This was from #4086